azerothcore/azerothcore-wotlk
# fix(DB/creature_loot): Crimson Hammersmith

## Changes Proposed:
- Adds a small chance for lvl appropriate greens (RLT 24016)
- Adds a small chance for lvl appropriate grays (RLT 24024)
- Adds Righteous orb, Major Healing Potion, Alterac Swiss and Morning Glory Dew to his loot table

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8475
- Closes https://github.com/chromiecraft/chromiecraft/issues/1856

## SOURCE:

https://tbc.wowhead.com/npc=11120/crimson-hammersmith
https://web.archive.org/web/20110613080354/http://www.wowhead.com:80/item=12811
Also look into the issue please

## Tests Performed:

- Ran SQL, no errors
- checked db
- also checked in-game (I added a temporary guid for Crimson Hammersmith and killed him multiple times)
- Notice that the changes for the loot here are very small, so most loots will only contain a few silver and the Bottom Half of Advanced Armorsmithing: Volume II (if you don't already have it in the inventory)

## How to Test the Changes:


1. Either check db
2. or go to Stratholme - Main Gate and pick up the BS plans (`.go o 42888`)
3. kill him/her multiple times and look at his loot

## Known Issues and TODO List:


- This is by no means a perfect loot table with a bulletproof source, but the best I could do with what we have.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
